tfsec fall in panic against directory which have same name modules

**Describe the bug**

When I run tfsec against following directory, tfsec fall in panic.

- Directory have some terraform roots.
- Each terraform root have reference to module and each reference's name is same.
- Module have `count` statement.

**To Reproduce**

```
$ git clone https://github.com/shibataka000/tfsec-issue
$ tfsec tfsec-issue
```

**Expected behavior**

tfsec should not fall in panic, and it should show result correctly.

**Screenshots**

```
$ tfsec tfsec-issue
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x69bdbe]

goroutine 1 [running]:
github.com/hashicorp/hcl/v2.(*EvalContext).Parent(...)
        /home/runner/work/tfsec/tfsec/vendor/github.com/hashicorp/hcl/v2/eval_context.go:24
github.com/aquasecurity/tfsec/internal/app/tfsec/block.(*HCLBlock).Clone(0xc0001a9840, 0x0, 0xaee500, 0xc0001fb8f0)
        /home/runner/work/tfsec/tfsec/internal/app/tfsec/block/hclblock.go:29 +0x3e
github.com/aquasecurity/tfsec/internal/app/tfsec/parser.(*Evaluator).expandBlockCounts(0xc0000f3900, 0xc0001a2400, 0x11, 0x20, 0x1, 0x1, 0xc0001a2400)
        /home/runner/work/tfsec/tfsec/internal/app/tfsec/parser/evaluator.go:226 +0x2a6
github.com/aquasecurity/tfsec/internal/app/tfsec/parser.(*Evaluator).EvaluateAll(0xc0000f3900, 0x2a, 0xc0002129c0, 0x2a, 0xc0000f3700, 0x5)
        /home/runner/work/tfsec/tfsec/internal/app/tfsec/parser/evaluator.go:178 +0x347
github.com/aquasecurity/tfsec/internal/app/tfsec/parser.(*Parser).ParseDirectory(0xc0001fa5a0, 0x1b, 0xc0000ba778, 0x1, 0x1, 0xc0001fa5a0)
        /home/runner/work/tfsec/tfsec/internal/app/tfsec/parser/parser.go:107 +0xb0b
main.glob..func2(0xfd2ac0, 0xc0001b38d0, 0x1, 0x1, 0x0, 0x0)
        /home/runner/work/tfsec/tfsec/cmd/tfsec/main.go:207 +0x69e
github.com/spf13/cobra.(*Command).execute(0xfd2ac0, 0xc0000c0010, 0x1, 0x1, 0xfd2ac0, 0xc0000c0010)
        /home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xfd2ac0, 0xc00004c778, 0xc0002dff78, 0x406365)
        /home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:902
main.main()
        /home/runner/work/tfsec/tfsec/cmd/tfsec/main.go:85 +0x31
```

**Desktop (please complete the following information):**
 
- OS: Ubuntu 20.04
- tfsec 0.48.2

**Additional context**

tfsec 0.47.2 or earlier doesn't fall in panic when I do above steps.

--- a/internal/app/tfsec/block/hclblock.go
+++ b/internal/app/tfsec/block/hclblock.go
@@ -26,7 +26,13 @@ func NewHCLBlock(hclBlock *hcl.Block, ctx *hcl.EvalContext, moduleBlock Block) B
 }
 
 func (block *HCLBlock) Clone(index int) Block {
-	childCtx := block.evalContext.Parent().NewChild()
+	var childCtx *hcl.EvalContext
+	if block.evalContext != nil {
+		childCtx = block.evalContext.NewChild()
+	} else {
+		childCtx = &hcl.EvalContext{}
+	}
+
 	if childCtx.Variables == nil {
 		childCtx.Variables = make(map[string]cty.Value)
 	}

